documentation-site/static/examples/heading/basic.js

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import {Heading, HeadingLevel} from 'baseui/heading';
+
+export default () => (
+  <HeadingLevel>
+    <Heading>Base Web [L1]</Heading>
```

Comment:
should we extend this example with a paragraph of Lorem ipsum text? To make it super clear how it works

Reply:
Added.